### ISSUE TITLE
crux-sql returns dates in default time zone

### ISSUE THREAD

**Issue by hraberg** [xtdb/xtdb#1080]
See:
```
lein test :only crux.calcite-test/test-different-data-types

FAIL in (test-different-data-types) (calcite_test.clj:463)
expected: (= [{:id ":human/ivan", :name "Ivan", :born born, :afloat afloat, :adecimal 1.3M}] (query "SELECT * FROM PERSON"))
  actual: (not (= [{:id ":human/ivan", :name "Ivan", :born #inst "2010-01-01T00:00:00.000-00:00", :afloat 1.0, :adecimal 1.3M}] [{:id ":human/ivan", :name "Ivan", :born #inst "2009-12-31T23:00:00.000000000-00:00", :afloat 1.0, :adecimal 1.3M}]))
```

The date is stored in UTC inside Crux, as can be seen via a normal Datalog query, so it gets turned into the wrong timezone (CEST here) somewhere inside crux-sql or Calcite.

**Comment by jarohen** [comment]
~Closed by #1079~

see https://github.com/juxt/crux/pull/1079/commits/0ca667a4329bc8e663321da278093ce47bfd710d

**Comment by jarohen** [comment]
repro'd: `(java.util.TimeZone/setDefault (java.util.TimeZone/getTimeZone "Asia/Kolkata"))`
